e2e_deployment_files/nestededge_middleLayerBaseDeployment_amqp.json

```diff
@@ -29,15 +29,12 @@
               "experimentalFeatures__enabled": {
                 "value": "true"
               },
-              "experimentalFeatures__nestedEdgeEnabled": {
-                "value": "true"
-              },
```

Comment:
what about the other deployment files?
- nestededge_isa95_smoke_test_BaseDeployment.json
- nestededge_middleLayerBaseDeployment_mqtt.json
- nestededge_topLayerBaseDeployment_amqp.json

Reply:
good catch. These have been added after my initial PR :)